### DISCUSSION TITLE
Any Plan to add One Way Broadcast

### DISCUSSION THREAD

**Discussion by hackstarsj** [mrgnr/camus#30] [Ideas]
Any Plan to add One Way Broadcast for Audio and Video or Can you help me to add this Feature too?

**Comment by mrgnr** [comment]
@hackstarsj I don't currently have plans to implement this feature since it would require significant architectural changes. To support this, the audio/video from the broadcaster would need to be streamed to the server and forwarded to other clients from there. At the moment, all media streams are peer-to-peer.
